terraform/aks/databases.tf

```diff
@@ -31,3 +31,22 @@ module "redis-queue" {
   azure_maxmemory_policy  = "noeviction"
 
 }
+
+module "postgres" {
+  source = "git::https://github.com/DFE-Digital/terraform-modules.git//aks/postgres?ref=aks-postgres"
+
+  namespace             = var.namespace
+  environment           = local.app_name_suffix
+  azure_resource_prefix = var.azure_resource_prefix
+  service_short         = var.service_short
+  config_short          = var.config_short
+
+  cluster_configuration_map = module.cluster_data.configuration_map
+
+  admin_username = local.infra_secrets.POSTGRES_ADMIN_USERNAME
+  admin_password = local.infra_secrets.POSTGRES_ADMIN_PASSWORD
```

Comment:
Could we let the module generate its own password instead? Similar to https://github.com/DFE-Digital/apply-for-qualified-teacher-status/pull/1358/files#diff-c18fcfa65aafd378a882c47eefd7555b9aeab967ab89fef20a5ffad5123981b9R26